.changeset/soft-moose-nail.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-api-specs/test': patch
+'@commercetools-docs/gatsby-theme-api-docs': patch
+'@commercetools-website/api-docs-smoke-test': patch
```

Comment:
known by now:  Only provide changesets for published packages.  I.e. only those in `@commercetools-docs/`

Reply:
Right, sorry for that 👍🏼